### ISSUE TITLE
Cannot start parity - "curl" binary not found

### ISSUE THREAD

**Issue by cleanunicorn** [vrde/ethnode#5]
### Description

Running `ethnode parity` fails with error.

```sh-session
$ ethnode parity
Download latest parity version, please wait.
"curl" binary not found, please install and retry
Error downloading parity, this might be temporary, try again later.
$ which curl
/usr/bin/curl
$ curl --version
curl 7.64.1 (x86_64-pc-linux-gnu) libcurl/7.64.1 OpenSSL/1.1.1b zlib/1.2.11 libidn2/2.1.1 libpsl/0.20.2 (+libidn2/2.1.1) libssh2/1.8.1 nghttp2/1.36.0
Release-Date: 2019-03-27
Protocols: dict file ftp ftps gopher http https imap imaps pop3 pop3s rtsp scp sftp smb smbs smtp smtps telnet tftp 
Features: AsynchDNS GSS-API HTTP2 HTTPS-proxy IDN IPv6 Kerberos Largefile libz NTLM NTLM_WB PSL SPNEGO SSL TLS-SRP UnixSockets
```

Version used:
```sh-session
$ ethnode --version
0.0.17
```

Parity already is available in my path, meaning I can run `parity` to connect to the main Ethereum network. I also have `geth` in my path.

My operating system is Arch Linux:
```sh-session
$ uname -a                    
Linux cola 5.0.9-arch1-1-ARCH #1 SMP PREEMPT Sat Apr 20 15:00:46 UTC 2019 x86_64 GNU/Linux
```

Current shell zsh:
```sh-session
$ echo $SHELL              
/usr/bin/zsh
$ zsh --version      
zsh 5.7.1 (x86_64-pc-linux-gnu)
```

**Comment by vrde** [comment]
Hi @cleanunicorn, thanks for reporting your issue. When `ethnode` starts for the first time, it downloads a new binary (geth or parity) and place it in a cache directory. The script to download parity is a ripoff of this one https://get.parity.io/, so if the original one works, mine should work as well :crossed_fingers:, but I see this is not what's happening here.

Can I ask you to run the install script manually? Here you are basically triggering the download of parity by hand:
```
git clone https://github.com/vrde/ethnode.git
cd ethnode
HOMEDIR=/tmp ./get_parity.sh
```
If it still fails, try removing from line 92 to line 98 in `get_parity.sh` with your editor or with:
```
sed -i -e '92,98d' get_parity.sh
```
and then run again:
```
HOMEDIR=/tmp ./get_parity.sh
```

You can also force the download with `ethnode -d parity`

**Comment by cleanunicorn** [comment]
Hello @vrde! 

The parity download scrips works well and as I said, I already have parity available in the path.

```sh-session
╭─daniel@cola /tmp ‹ruby-2.5.3› 
╰─$ ./parity_get.sh 
Release selected is: beta
Upgrading parity from 2.4.6 to 2.5.1
[sudo] password for daniel: 
╭─daniel@cola /tmp ‹ruby-2.5.3› 
╰─$ parity --version                                                                                  
Parity Ethereum
  version Parity-Ethereum/v2.5.1-beta-adabd81-20190514/x86_64-linux-gnu/rustc1.34.1
Copyright 2015-2019 Parity Technologies (UK) Ltd.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

By Wood/Paronyan/Kotewicz/Drwięga/Volf
   Habermeier/Czaban/Greeff/Gotchac/Redmann

╭─daniel@cola /tmp ‹ruby-2.5.3› 
╰─$ ethnode parity   
Download latest parity version, please wait.
"curl" binary not found, please install and retry
Error downloading parity, this might be temporary, try again later.
```

**Comment by vrde** [comment]
The problem should be in: `"curl" binary not found, please install and retry`. Don't know the reason why it is not available when running `ethnode parity`, while `curl` is available if you run `get_parity.sh` manually :thinking:

I've just released version `0.0.18` but it shouldn't make any difference for this issue.

I've intsalled ethnode successfully on other GNU/Linux machines, and also on Mac. I'd like to help you more, but I don't have ideas for now.
